samples/Microsoft.AspNetCore.Proxy.Samples/Microsoft.AspNetCore.Proxy.Samples.csproj

```diff
@@ -13,4 +13,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="$(AspNetCoreVersion)" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
```

Comment:
Don't check a specific target version 😄 
Check the target framework the same way you do here https://github.com/aspnet/Proxy/blob/c48f03437605a1f87804ceedbab12b09fa9d38ed/build/common.props#L19-L21